src/GitHub.VisualStudio/UI/Views/GitHubPaneViewModel.cs

```diff
@@ -38,6 +39,9 @@ public class GitHubPaneViewModel : TeamExplorerItemBase, IGitHubPaneViewModel
         Microsoft.VisualStudio.Shell.OleMenuCommand back, forward, refresh;
         int latestReloadCallId;
 
+        [Import]
+        IVisualStudioBrowser VisualStudioBrowser { get; set; }
```

Comment:
The rest of the services in this class are imported via the constructor, so probably best to do that for this too.